marlowe-chain-sync/src/Language/Marlowe/Runtime/Cardano/Api.hs

```diff
@@ -251,7 +251,7 @@ assetsToCardanoValue Assets{..} = fold <$> sequence
 
 assetsFromCardanoValue :: C.Value -> Assets
 assetsFromCardanoValue value = Assets
-  { ada = maybe 0 fromCardanoLovelace $ C.valueToLovelace value
+  { ada = fromCardanoLovelace $ C.selectLovelace value
```

Comment:
Caught a bug! `valueToLovelace` returns `Just` if and only if the `value` is ADA-only.